src/Symfony/Component/Serializer/Encoder/ContextAwareDecoderInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Encoder;
+
+/**
+ * Adds the support of an extra $context parameter for the supportsDecoding method.
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+interface ContextAwareDecoderInterface extends DecoderInterface
```

Comment:
[I don't think it can work by extending the old interface](https://3v4l.org/fjjMK) or am I missing something ? 😕